talon.wdl

```diff
@@ -446,39 +446,38 @@ task SummarizeDatasets {
 
 task Talon {
     input {
-        File SAMfile
```

Comment:
SAM files are specified in the config files right? How are they getting localized now?

Side question, can we use BAM files instead?

Reply:
Correct, they are specified in the config file. I now generate the config file in the main TALON-WDL pipeline. It uses the SAM files in the specified output directory.

Reply:
Sadly BAM has not yet been implemented. pysam is now being used for the parsing of the SAM file, but the BAM option is not yet provided, we might have to implement that ourselves at some point.

Reply:
SAM files still need to get localized to the task, they should still be provided as an input.

Reply:
But they wouldn't be the same SAM files as in the config file in that case, so TALON wouldn't use them. Unless I alter or create the config file in the talon task. Would that be preferable to creating it in a separate task in the pipeline.wdl of TALON-WDL?

Reply:
Better to make the config in the same task. Right now, this task will only work if a custom configuration is used, which mounts whatever directory the SAM files are in.

Reply:
TALON wants the absolute path to the SAM files, or it has to be in the same folder as the TALON process is running from (I think thats how I used to make it work with the mv's, but it was for single SAM file runs).

Reply:
I believe cromwell will make them absolute paths if they are of type `File`

Reply:
I have now included some bash code to create the config file.